charms/kfp-persistence/requirements.in

```diff
@@ -1,3 +1,4 @@
+charmed-kubeflow-chisme @ git+https://github.com/canonical/charmed-kubeflow-chisme.git@add-base-charm
 ops
 oci-image
```

Comment:
After yesterday's conversation, I was inspired to raise that (I think) we don't need oci-image anymore. Maybe this goes for `requirements-unit.in` as well?

Reply:
Will remove.

Reply:
still here :)